_includes/projects.html

```diff
@@ -206,4 +206,27 @@ <h5 class="font-weight-bold text-white bg-purple mb-0 text-center">{{ item.proje
             </div>
         </div>
     </div>
+    <div class="container">
+        <div class="row">
+            <div class="col-12 col-sm-2 text-center bg-blue text-white d-flex flex-column justify-content-center align-items-center">
+                <h4 class="">OpenHPC</h4>
```

Comment:
Shouldn't this be `HPC` ?
Many of the projects in `category: hpc` below are not components in OpenHPC.